tests/core/pyspec/eth2spec/test/helpers/attestations.py

```diff
@@ -180,11 +173,16 @@ def fill_aggregate_attestation(spec, state, attestation, committee_index, signed
     if filter_participant_set is not None:
         participants = filter_participant_set(participants)
 
+    # initialize `aggregation_bits`
     if is_post_eip7549(spec):
-        attestation.committee_bits = spec.Bitvector[spec.MAX_COMMITTEES_PER_SLOT]()
```

Comment:
note: already initialized to "empty" type which is equivalent here for `Bitvector`